lib/game.rb

```diff
@@ -0,0 +1,54 @@
+require '../lib/interface.rb'
+
+class Game
+  include Interface
+  attr_accessor :symbol
+
+  def initialize(board, player_1, player_2)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+    @turn = 1
+  end
+
+  def gameplay
```

Comment:
I was thinking of a form for this method that goes something like:

```
def gameplay
  until game_over?
    action(player_1)
    action(player_2)
  end
end
```
So that we won't need to use an `if else` block. This makes the method much simpler to understand (and much shorter). Can you refactor your logic a bit so that this gameplay method will be similar to what I wrote above?